client/src/components/App/Content/LandingPage/adminLandingPage/adminActions/adminMessages/adminMessages.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
+import { Grid, Typography, Collapse } from '@material-ui/core';
+
+import Message from '../message/message';
+import useStyles from './adminMessegesStyles';
+import StoreStateType from 'redux/storeStateType';
+import { AdminMessage } from 'models/AdminMessage';
+import useAdminMessagesDBAction from './useAdminMessagesDBAction';
+import Desk from 'models/Desk';
+
+const AdminMessages: React.FC<Props> = (props: Props): JSX.Element => {
+  const classes = useStyles();
+  const { getAdminsMessagesByAdmin, adminsMessagesByAdmin, sendMessage, deleteMessage, toRefresh } = useAdminMessagesDBAction();
+
+  const messageTitle = 'הודעה';
+  const messageDescriptionTitle = '(תוצג בטבלת הדסק המסומן בעמוד זה)';
```

Comment:
Is it the right text?

Reply:
https://dev.azure.com/spectrumFactory/c29f56f9-1895-4190-94f9-3d13819ee54a/_apis/wit/attachments/452399c3-588f-43e8-951c-20b3d341e0f2?fileName=image.png
yes...